gnuboardProject/gnuboardProject/frontend/src/components/Portfolio/PortfolioForm.tsx

```
@@ -130,13 +236,15 @@ const PortfolioForm: React.FC = () => {
         <label>섹션</label>
-        {sections.map((section, idx) => (
-          <div key={idx} style={{border:'1px solid #eee',borderRadius:8,padding:'1rem',marginBottom:'1rem'}}>
-            <input value={section.title} onChange={e=>handleSectionChange(idx,'title',e.target.value)} placeholder="섹션 제목" style={{width:'100%',marginBottom:8,padding:'0.5rem',border:'1px solid #ddd',borderRadius:6}} />
-            <textarea value={section.content} onChange={e=>handleSectionChange(idx,'content',e.target.value)} placeholder="섹션 내용" style={{width:'100%',padding:'0.5rem',border:'1px solid #ddd',borderRadius:6}} />
-            {sections.length > 1 && <button type="button" onClick={()=>removeSection(idx)} style={{marginTop:8,color:'#dc3545',background:'none',border:'none',cursor:'pointer'}}>삭제</button>}
-          </div>
-        ))}
+        <TemplatePreview template={template}>
+          {sections.map((section, idx) => (
+            <div key={idx} className="section">
+              <input value={section.title} onChange={e=>handleSectionChange(idx,'title',e.target.value)} placeholder="섹션 제목" style={{width:'100%',marginBottom:8,padding:'0.5rem',border:'1px solid #ddd',borderRadius:6}} />
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)